src/main/java/core/basesyntax/model/User.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.model;
+
+public class User {
+    private String name;
+
+    User(String name) {
+        this.name = checkName(name);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    private String checkName(String name) {
+        if (name == null || name.endsWith("")) {
+            throw new RuntimeException("Wrong name");
+        }
+        return name;
+    }
```

Comment:
let's remove this
the purpose of name.endsWith("") is not clear
and the name won't be null because the user will enter it to the console

Reply:
done